[otlp] Remove Experimental Flag and emit stable Exception attributes by default (#5258)
open-telemetry/opentelemetry-dotnet: src/OpenTelemetry.Exporter.OpenTelemetryProtocol/CHANGELOG.md

  `LoggerProviderBuilder.AddOtlpExporter` registration extensions.
  [#5103](https://github.com/open-telemetry/opentelemetry-dotnet/pull/5103)

* Removed the `OTEL_DOTNET_EXPERIMENTAL_OTLP_EMIT_EXCEPTION_LOG_ATTRIBUTES`
  environment variable, following the stabilization of the exception attributes
  `exception.type`, `exception.message`, and `exception.stacktrace` in the
  [OpenTelemetry Semantic
  Conventions](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/exceptions/exceptions-logs.md#semantic-conventions-for-exceptions-in-logs).

>   These attributes, corresponding to `LogRecord.Exception`, are now standard and
>   will be automatically included in exports. The need for the previously
>   experimental flag is thus eliminated.

Reviewer: ```suggestion
  These attributes, corresponding to `LogRecord.Exception`, are now stable and
  will be automatically included in exports.
```
